src/ImageSharp/Metadata/Profiles/Exif/ExifConstants.cs

```diff
@@ -2,11 +2,25 @@
 // Licensed under the Apache License, Version 2.0.
 
 using System;
+using System.Buffers.Binary;
+using System.Text;
 
 namespace SixLabors.ImageSharp.Metadata.Profiles.Exif
 {
     internal static class ExifConstants
     {
+        private const ulong AsciiCode = 0x_41_53_43_49_49_00_00_00;
+        private const ulong JISCode = 0x_4A_49_53_00_00_00_00_00;
+        private const ulong UnicodeCode = 0x_55_4E_49_43_4F_44_45_00;
+        private const ulong UndefinedCode = 0x_00_00_00_00_00_00_00_00;
+
+        private static readonly byte[] AsciiCodeBytes = { 0x41, 0x53, 0x43, 0x49, 0x49, 0, 0, 0 };
```

Comment:
```suggestion
        // This uses C# compiler's optimization to refer to the data segment directly. No allocation occurs.
        private static ReadOnlySpan<byte> AsciiCodeBytes => new byte[] { 0x41, 0x53, 0x43, 0x49, 0x49, 0, 0, 0 };
```
With the `byte[]` the runtime needs to allocate an array on the heap. With the ROS<byte> the C# compiler will do a trick, and refer to the assembly's static data segment without any further allocation.

This is already used in other places in this project (e.g. L24 below), so should be used here too.